### PR TITLE
fix: increase test coverage to 80%

### PR DESCRIPTION
**Pull request by axthosarouris**
Ulitmate goal: 100% test coverage. 

The 100% test coverage is going to be a achieved in two PRs. 
First PR (this one) goal is to reach 80% test coverage. 

Second PR goal will be to reach 100% test coverage

### CODE REVIEW

File: src/main/java/nva/commons/exceptions/FileNotFoundUncheckedException.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package nva.commons.exceptions;
+
+import java.nio.file.Path;
+
+public class FileNotFoundUncheckedException extends RuntimeException {
+
+    public static final String FILE_NOT_FOUND_MESSAGE = "File not found:";
```
Comment (brinxmat) [inline src/main/java/nva/commons/exceptions/FileNotFoundUncheckedException.java:7]: Space after colon

File: src/main/java/nva/commons/exceptions/ResourceNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package nva.commons.exceptions;
+
+import java.nio.file.Path;
+
+public class ResourceNotFoundException extends RuntimeException {
+
+    public static final String ERROR_MESSAGE = "Could not find resource in path:";
```
Comment (brinxmat) [inline src/main/java/nva/commons/exceptions/ResourceNotFoundException.java:7]: Space after colon

File: src/main/java/nva/commons/hanlders/ApiMessageParser.java [hunk 3]
```diff
@@ -16,10 +18,21 @@
  */
 public class ApiMessageParser<T> {
 
-    private final transient ObjectMapper mapper = JsonUtils.jsonParser;
+    public static final String COULD_NOT_PARSE_REQUEST_INFO = "Could not parse RequestInfo";
```
Comment (brinxmat) [inline src/main/java/nva/commons/hanlders/ApiMessageParser.java:21]: Is it an idea to dump the json of the request info? If there are no secrets

Comment (axthosarouris) [reply src/main/java/nva/commons/hanlders/ApiMessageParser.java:21]: It has the Bearer token. But anything else could be sent to the response body.

**Pull request merged**